pg_search/src/postgres/build.rs

```diff
@@ -263,7 +263,7 @@ fn do_heap_scan<'a>(
     state
 }
 
-#[cfg(any(feature = "pg13", feature = "pg14", feature = "pg15", feature = "pg16"))]
+#[cfg(any(feature = "pg13", feature = "pg14", feature = "pg15", feature = "pg16", feature = "pg17"))]
```

Comment:
We can remove this `#[cfg` since we deprecated PG12. 13-17 is all the versions we support

Reply:
Rustfmt is blocking due to these `cfg` macros, so I'm trying to remove them in this PR directly.